lib/modules/datasource/github-runners/index.ts

```diff
@@ -12,8 +12,10 @@ export class GithubRunnersDatasource extends Datasource {
     ubuntu: [{ version: '22.04' }, { version: '20.04' }, { version: '18.04' }],
     macos: [
       { version: '13' },
+      { version: '13-l' },
       { version: '13-xl' },
       { version: '12' },
+      { version: '12-l' },
       { version: '12-xl' },
```

Comment:
```suggestion
      { version: '12-large' },
```
`macos-12-xlarge` doesn't exist